lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/device/connection/DeviceConnectionManager.java

```diff
@@ -125,6 +130,25 @@ Establish connection with device (future will be set with GnmiDeviceManager for
             executorService);
     }
 
+    private Gnmi.CapabilityResponse getExtensionYangModelsList(final DeviceConnection deviceConnection) {
+        Optional<Map<AdditionalCapabilityKey, AdditionalCapability>> models =
+            deviceConnection.getConfigurableParameters().getAdditionalCapabilities();
+        Gnmi.CapabilityResponse.Builder capabilitiesResponseBuilder = Gnmi.CapabilityResponse.newBuilder();
+        if (models.isPresent()) {
+            models.get().entrySet()
+                .stream()
+                .map(model -> model.getValue())
+                .peek(additionalCapability ->
+                    LOG.debug("Adding additional gNMI capability: name {} version {}",
```

Comment:
this is not actually the adding of capability to capabilities list used for node, it should not be stated like this before it was actually done, there are multiple places where it can fail after this log, so the capability may not be added actually